modules/bim/app/seeders/bim/basic_data/theme_seeder.rb

```diff
@@ -39,6 +39,10 @@ def seed_data!
           .new(theme)
           .call
       end
+
+      def applicable?
+        !DesignColor.any?
```

Comment:
```suggestion
        DesignColor.empty?
```

Reply:
IMHO `empty?` is easier to read than the negation bang way.

https://api.rubyonrails.org/v4.1.0/classes/ActiveRecord/Relation.html#method-i-empty-3F

Reply:
Yes, I agree. Will change that